BenadrylAutomation/src/main/java/com/jnj/pageobjects/SignUpPage.java

```diff
@@ -171,4 +159,16 @@ public void checkLinks(String key, String expectedUrl) {
 			Assert.fail("Link not displayed");
 		}
 	}
+	
+	/**
+	 * Function to close signup lightbox
+	 */
+	public void closeSignUpPopup() {
```

Comment:
@goutamnaik007 this function you added at the bottom from line 81 ?

Reply:
yes. After merge there were two copies of the same. So, I removed one of it from the middle
@vaibhavNagvekarSJI